package.json

```diff
@@ -1,86 +1,65 @@
 {
   "name": "r7insight_node",
-  "version": "1.8.3",
-  "description": "Logentries client for node; use with or without Winston or Bunyan.",
+  "version": "2.0.0",
+  "description": "Rapid7 Insight Platform client for node; use with or without Winston or Bunyan.",
   "keywords": [
     "logentries",
     "logentries.com",
     "logs",
     "logging",
     "winston",
-    "bunyan"
+    "bunyan",
+    "insightops",
+    "rapid7",
+    "rapid7.com",
+    "insightplatform",
+    "insight"
   ],
   "homepage": "https://github.com/rapid7/r7insight_node",
   "author": {
-    "name": "Darien Maillet Valentine",
-    "email": "darien.valentine@logentries.com"
+    "name": "Rapid7",
+    "email": "InsightOpsTeam@rapid7.com"
   },
-  "contributors": [
-    {
-      "name": "Darien Maillet Valentine",
-      "email": "darien.valentine@logentries.com"
-    },
-    {
-      "name": "Paul Dixon",
-      "email": "paul.dixon@logentries.com"
-    }
-  ],
   "license": "BSD-3-Clause",
   "repository": {
     "type": "git",
     "url": "https://github.com/rapid7/r7insight_node.git"
   },
-  "main": "lib/logger.js",
+  "main": "src/index.js",
   "scripts": {
-    "build": "gobble build lib --force",
     "lint": "eslint src",
-    "prepublish": "npm run build",
-    "test": "npm run build && npm run lint && node test/test.js | tap-spec"
+    "lint:fix": "eslint src --fix",
+    "test": "npm run lint && node test/test.js | tap-spec",
+    "test:debug": "node test/test.js | tap-spec"
   },
   "engines": {
-    "iojs": ">=0.10",
-    "node": ">=0.8.0",
-    "npm": ">=1.4.6"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
   },
   "dependencies": {
-    "babel-runtime": "6.6.1",
-    "codependency": "0.1.4",
-    "json-stringify-safe": "5.0.1",
-    "lodash": "4.17.12",
-    "semver": "5.1.0",
-    "reconnect-core": "1.3.0"
+    "codependency": "^2.1.0",
+    "json-stringify-safe": "^5.0.1",
+    "lodash": "^4.17.15",
+    "reconnect-core": "^1.3.0"
   },
   "devDependencies": {
-    "babel": "6.5.2",
-    "babel-plugin-add-module-exports": "0.1.2",
-    "babel-plugin-syntax-decorators": "6.5.0",
-    "babel-plugin-transform-decorators-legacy": "1.3.4",
-    "babel-plugin-transform-runtime": "6.6.0",
-    "babel-preset-es2015": "6.6.0",
-    "babel-preset-stage-3": "6.5.0",
-    "bunyan": "1.8.4",
-    "eslint": "3.3.0",
-    "eslint-config-airbnb": "10.0.0",
-    "eslint-plugin-import": "1.12.0",
-    "eslint-plugin-jsx-a11y": "2.0.1",
-    "eslint-plugin-react": "6.0.0",
-    "estraverse": "4.2.0",
-    "estraverse-fb": "1.3.1",
-    "gobble": "0.10.2",
-    "gobble-babel": "6.0.0",
-    "gobble-cli": "0.6.0",
-    "install": "0.6.1",
-    "mitm": "1.3.2",
-    "npm": "3.8.2",
-    "tap-spec": "3.0.0",
-    "tape": "4.0.0",
-    "winston": "1.0.0",
-    "winston1": "1.1.2",
-    "winston2x": "2.1.1"
+    "bunyan": "^1.8.12",
+    "eslint": "^6.6.0",
+    "eslint-config-airbnb": "^18.0.1",
```

Comment:
Can you replace this with the Rapid7 one :)

Reply:
yessir